.gitlab/pipeline/build.yml

```diff
@@ -114,11 +114,30 @@ build-rustdoc:
   script:
     # FIXME: it fails with `RUSTDOCFLAGS="-Dwarnings"` and `--all-features`
     # FIXME: return to stable when https://github.com/rust-lang/rust/issues/96937 gets into stable
-    - time cargo doc --workspace --no-deps
+    - time cargo doc --features try-runtime,experimental --workspace --no-deps
     - rm -f ./target/doc/.lock
     - mv ./target/doc ./crate-docs
     # FIXME: remove me after CI image gets nonroot
     - chown -R nonroot:nonroot ./crate-docs
+    # Inject Simple Analytics (https://www.simpleanalytics.com/) privacy preserving tracker into
+    # all .html files
+    - |
+      inject_simple_analytics() {
```

Comment:
I dont see a CI scripts folder anymore, we used to have one :thinking:

Reply:
It was a little bit messy so I decided to put files and scripts related to gitlab to `.gitlab` folder and the same for github actions.